### ISSUE TITLE
Missing attr_accessors in Blob class

### ISSUE THREAD

**Issue by aaronrussell-x** [fcoury/octopi#33]
I've been getting errors using the Blob class when sending the :path attribute ("no attr_accessor set for xxx on Blob"). I believe you should have all the following attr_accessors:

```
attr_accessor :text, :data, :name, :sha, :size, :mode, :mime_type
```

**Comment by radar** [comment]
Hi Aaron,

What is the code to duplicate this issue? As far as I can see, blob data is returned as a string from Github.

**Comment by radar** [comment]
Add more attr_accessors to Blob - Closed by 237e416520fe934b29a752d0529049fe120f33c7
